### DISCUSSION TITLE
DASH Streaming videos consistently fail to download

### DISCUSSION THREAD

**Discussion by foush** [aclap-dev/video-downloadhelper#322] [Help and Q&A]
### Your question

I'm attempting to download a video, the download progresses slowly until it reaches 99% where it suddenly fails with the following message:

Native port disconnected

moz-extension://6e6d0cbf-8f17-455a-81e9-9a57b6556d3b/background/main.js:1 cleanup@moz-extension://6e6d0cbf-8f17-455a-81e9-9a57b6556d3b/background/main.js:1:43568 nd</callCatchAppNotFound/</<@moz-extension://6e6d0cbf-8f17-455a-81e9-9a57b6556d3b/background/main.js:1:42725 

using Chrome for the same video, I get a similar error:
Error: Native port disconnected at Ss.cleanup (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:43566) at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42723
Note in both chrome and firefox I'm able to download other videos from the site, it seems to only be for DASH streaming video downloads

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): 89a7b500b39446e5
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/foush",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by Mattmjs** [comment]
Is there a link to share or is it private ? otherwise post the hit details which include master & media manifests.

**Reply by foush** [reply]
Unfortunately it's a private site so downloads are per-authed-user session
